swagger.yml

```diff
@@ -5022,7 +5022,23 @@ paths:
           name: older-than
   '/hosts/{hostname}/technologies/all/databases/grant-dba':
     get:
-      summary: Search oracle dba grants
+      summary: Search oracle dba grants by hostname
```

Comment:
https://editor.swagger.io/ gives these errors:
 
Semantic error at paths./hosts/{hostname}/technologies/all/databases/grant-dba
Declared path parameter "hostname" needs to be defined as a path parameter at either the path or operation level
Jump to line 5023

Semantic error at paths./hosts/technologies/all/databases/grant-dba.get.operationId
Operations must have unique operationIds.
Jump to line 5054

Could you fix them?